docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,20 @@
+# Release Process
+KFServing's automated release processes run in Google Cloud Build under the project `kfserving`. For additional permissions to this project, please contact ellisbigelow@google.com.
+
+Builds are available at https://console.cloud.google.com/cloud-build/builds?project=kfserving.
+To view builds, you must be a member of kubeflow-discuss@googlegroups.com.
+
+## Latest Release
+This release process relies on Google Cloud build Triggers. This build triggers are configured manually to execute builds for the following:
+
+1. github.com/kubeflow/kfserving/Dockerfile -> `gcr.io/kfserving/kfserving-controller:latest`
+2. github.com/kubeflow/kfserving/python/sklearn.Dockerfile -> `gcr.io/kfserving/sklearnserver:latest`
```

Comment:
How about rename to `sklearnserving` and `xgbserving` to be consistent with `tfserving` and `kfserving`?

Reply:
we can maybe call sklserving and xgbserving - but has to be 3 letter minimum for them.

Reply:
I'm nervous about going down this road. I don't think it generalizes to mxnet, tensorrt, pytorch.

I'd hate to make a box we try to shove everything into:
mxnserving
xgbserving
sklserving
trtserving
tfserving

:(:(

Reply:
To be clear, I aligned with our folder names.

Reply:
If we want to rename these, it's significant effort. I will punt on this.